app/models/doi.rb

```diff
@@ -1273,30 +1273,30 @@ def downloads_over_time
   end
 
   def reference_ids
-    reference_events.pluck(:target_doi).compact.uniq.map(&:downcase)
+    reference_events.pluck(:target_doi).compact.map(&:downcase).uniq
   end
 
   def reference_count
-    reference_events.pluck(:target_doi).uniq.length
+    reference_events.pluck(:target_doi).compact.map(&:downcase).uniq.length
```

Comment:
@codycooperross the only suggestion I would make is not exactly to do with your change but with the existing code repetition. This happens in a few places but if you look at the method reference_count it's code is exactly the same as reference_ids except for the additional length method invocation. Because we didn't reuse the code we had to make this change in two places. 

This is certainly outside the scope of your  PR. I will create an issue to clean up this code to make more DRY (don't repeat yourself).

I will approve this PR.

Reply:
Thank you, Wendel!